Sources/Bot/Services/MergeService.swift

```diff
@@ -543,7 +548,7 @@ extension MergeService {
                             return requiredStateProducer.zip(with: .value(pullRequestMetadataRefreshed))
                         }
                         .flatMapError { _ in .empty }
-                        .filterMap { state, pullRequestMetadataRefreshed in
+                        .compactMap { state, pullRequestMetadataRefreshed in
```

Comment:
Good stuff, I forgot to change this.

Reply:
@AliSoftware there's an additional `filterMap` that needs to be changed IIRC can you check and change it too?

Reply:
Just a warning but still.

Reply:
Good catch. f9ad7a4208d2c2cdb29ca12f7bcbe0cb908dfd3f